src/providers/app-config/app-config.ts

```diff
@@ -30,32 +30,26 @@ export class AppConfigProvider {
   }
 
   private getRemoteData = (): Observable<any> => {
-    return this.httpClient.get<any>(this.environmentFile.remoteSettingsUrl).map((res) => {
-      this.appConfig = {
-        googleAnalyticsId: res.body.data.googleAnalyticsId,
-        userIdDimensionIndex: res.body.data.userIdDimensionIndex,
-        authentication: {
-          context: res.body.data.authentication.context ,
-          redirectUrl:  res.body.data.authentication.redirectUrl,
-          resourceUrl: res.body.data.authentication.resourceUrl,
-          clientId: res.body.data.authentication.clientId,
-          logoutUrl: res.body.data.authentication.logoutUrl,
-        },
-      }
-      return;
+    return this.httpClient.get<any>(this.environmentFile.remoteSettingsUrl)
+    .map((res) => {
+      this.mapConfig(res.body.data);
     });
   }
 
   private getLocalData = () => {
+    this.mapConfig(this.environmentFile);
+  }
+
+  private mapConfig = (data: any) => {
     this.appConfig = {
```

Comment:
Do we have an interface for appConfig anywhere?

Reply:
We have a type - https://github.com/dvsa/mes-mobile-app/blob/develop/src/providers/app-config/app-config.model.ts